lib/widgets/action_sheet.dart

```diff
@@ -17,15 +17,26 @@ import 'store.dart';
 ///
 /// Must have a [MessageListPage] ancestor.
 void showMessageActionSheet({required BuildContext context, required Message message}) {
+  final store = PerAccountStoreWidget.of(context);
+
   // The UI that's conditioned on this won't live-update during this appearance
   // of the action sheet (we avoid calling composeBoxControllerOf in a build
   // method; see its doc). But currently it will be constant through the life of
   // any message list, so that's fine.
   final isComposeBoxOffered = MessageListPage.composeBoxControllerOf(context) != null;
+
+  final selfUserId = store.account.userId;
+  final hasThumbsUpReactionVote = message.reactions
+    ?.aggregated.any((reactionWithVotes) =>
+      reactionWithVotes.emojiCode == '1f44d'
```

Comment:
Should check `reactionType` before `emojiCode`, as the latter is namespaced by the former.